ckcp/openshift_dev_setup.sh

```diff
@@ -281,6 +281,30 @@ patches:
   KUBECONFIG_KCP="$WORK_DIR/credentials/kubeconfig/kcp/ckcp-ckcp.default.compute.kubeconfig"
 }
 
+install_rekor() {
+  # Get our domain from the ingresses config of the cluster
+  echo -n "  - Domain: "
+  domain=$(kubectl get ingresses.config.openshift.io cluster -o json|jq -r ".spec.domain")
```

Comment:
There is the same statement in the function `install_ckcp`([link](https://github.com/openshift-pipelines/pipelines-service/blob/main/ckcp/openshift_dev_setup.sh#L212)). To avoid the duplicate statement in anywhere, would you mind moving it into one place, such as `init` function? So that we can use `domain` variable in other places.

Reply:
Done